clone-repos/index.js

```diff
@@ -68,17 +68,20 @@ const cloneRepository = async (repositoryUrl, destinationPath) => {
 };
 
 async function cloneRepos(orgName) {
-  const repos = await getRepos(orgName);
-  for (let repo of repos) {
-    if (!ISOMER_ADMIN_REPOS.includes(repo.name)) {
+  // const repos = await getRepos(orgName);
```

Comment:
Are we keeping this line commented so that we can clone all sites if needed?

Reply:
But that will require code changes to use repo.name instead I guess?

Reply:
shucks you're right - have removed it in https://github.com/isomerpages/isomer-tooling/pull/53/commits/654da93be3708e532379833da9a36046a7250e02